templates/practice/documentation.html

```diff
@@ -8,6 +8,11 @@
 
 {% block meta_image %}https://assets.ubuntu.com/v1/253da317-image-document-ubuntudocs.svg{% endblock %}
 
+<!-- This should be removed when the page is ready so it can be indexed by search engines. -->
+<head>
+  <meta name="robots" content="noindex">
+</head>
+
```

Comment:
```suggestion
{% block extra_metatags %}
    <!-- This should be removed when the page is ready so it can be indexed by search engines. -->
    <meta name="robots" content="noindex">
{% endblock %}
```

Reply:
Awesome, cheers for this!